pdf/model/textencoding/glyphlist/utils/glyphparser/glyphparser.go

```diff
@@ -80,22 +93,61 @@ func printRuneToGlyphList(glyphToUnicodeMap map[string]string) {
 
 	uniqueList := map[string]bool{}
 
-	fmt.Printf("var glyphlistRuneToGlyphMap = map[rune]string{\n")
+	fmt.Printf("var glyphlistRuneToGlyphMap = map[rune]string{ // %d entries \n", len(keys))
 	for _, glyph := range keys {
-		ucode := glyphToUnicodeMap[glyph]
-		ucode = strings.ToLower(ucode)
-
-		_, duplicate := uniqueList[ucode]
+		s := glyphToUnicodeMap[glyph]
+		line := ""
+		runes := []rune(s)
+		if len(runes) > 1 {
+			line = fmt.Sprintf("%+q:\t%q, %s", s, glyph, ss(s))
+		} else {
+			if asRune {
+				r := runes[0]
+				line = fmt.Sprintf("%+q:\t%q, %s", r, glyph, rs(r))
+			} else {
+				line = fmt.Sprintf("%+q:\t%q, %s", s, glyph, ss(s))
+			}
+		}
+		_, duplicate := uniqueList[s]
 		if !duplicate {
-			fmt.Printf("\t'\\u%s':\t\"%s\",\n", ucode, glyph)
-			uniqueList[ucode] = true
+			fmt.Printf("\t\t%s\n", line)
+			uniqueList[s] = true
+		} else if len(runes) > 1 {
+			fmt.Printf("\t\t// %s -- ambiguous - multiple characters\n", line)
 		} else {
-			fmt.Printf("//\t'\\u%s':\t\"%s\", // duplicate\n", ucode, glyph)
+			fmt.Printf("\t\t// %s -- duplicate\n", line)
 		}
 	}
 	fmt.Printf("}\n")
 }
 
+// ss shows string u
+func ss(u string) string {
+	s := ""
+	printable := false
+	for _, r := range u {
+		if unicode.IsPrint(r) {
+			printable = true
+		}
+	}
+	if printable {
+		s = fmt.Sprintf("%#q", u)
+		s = fmt.Sprintf("// %s", s[1:len(s)-1])
+	}
+	return s
+}
+
+// ru returns a string with the Go code for rune `r` and, if the rune is printable, an comment
+// showing how it prints.
+func rs(r rune) string {
```

Comment:
ru or rs?

Reply:
More obvious names for methods ss and rs would be nice?  showString, showRune ?  or printString, printRune if this is a debugging function